orientation-event/ondeviceorientationabsolute.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>User agents must also provide an event handler IDL attribute named ondeviceorientationabsolute on the window object</title>
+    <meta name=viewport content="width=device-width, maximum-scale=1.0">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <p>Rotate the device to run the tests.</p>
```

Comment:
Is it a manual test?

Reply:
Nope. The spec says "If an implementation can never provide orientation information, the event should be fired with the alpha, beta and gamma attributes set to null." (which tested in a separate test).